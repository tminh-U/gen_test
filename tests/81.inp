5
16 11 9 15 16 
11
1 2
1 1
1 5
2 5
3 5
3 4
3 3
5 5
5 5
2 3
2 4
