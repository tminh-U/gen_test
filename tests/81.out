3
2
14
12
8
4
2
2
2
3
6
